commit c58cfcd039f5b932404ccde70e429e174bed2258
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-14T10:16:59-07:00 (4 months ago)
    
    TRITON-1748 'sdcadm experimental remove-ca' should trace log to file
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index f2c066e..3654916 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,8 @@
 
 # sdcadm Changelog
 
+- TRITON-1748 Ensure 'sdcadm experimental remove-ca' does trace logging to file.
+
 ## 1.30.0
 
 - TRITON-1319 need firewall-logger agent (`sdcadm post-setup firewall-logger-agent`)
@@ -688,7 +690,7 @@ external nic for IMGAPI
 
 ## 1.3.8
 
-- Add `sdcadm experimental install-platform` and `sdcadm expertimental assign-platform`
+- Add `sdcadm experimental install-platform` and `sdcadm experimental assign-platform`
 
 ## 1.3.7
 
diff --git a/lib/cli/do_remove_ca.js b/lib/cli/do_remove_ca.js
index e17d177..177261c 100644
--- a/lib/cli/do_remove_ca.js
+++ b/lib/cli/do_remove_ca.js
@@ -96,6 +96,7 @@ do_remove_ca.help = [
     'related service agents and VMs.'
 ].join('\n');
 
+do_remove_ca.logToFile = true;
 
 // --- exports
 
